### DISCUSSION TITLE
关于Linux中运行mirai-login-solver-selenium

### DISCUSSION THREAD

**Discussion by Zqysl** [mamoe/mirai#1107] [Q&A]
家里有个闲置的树莓派，打算用来挂机器人。目前卡在了滑块验证，迟迟得不到解决

情景复现：
1. 用MCL登录账号时，提示滑块验证，于是给MCL装载了mirai-login-solver-selenium，
2. 再次尝试登陆时，mirai-login-solver-selenium报错，提示在Linux中只支持FireFox，于是下载了FireFox并作为默认浏览器
3. 再次尝试登陆时，依旧会出现上述报错

有没有在Linux中成功运行mirai的大佬，希望可以分享一下经验

**Comment by Karlatemp** [comment]
手动完成验证 章节

**Reply by Zqysl** [reply]
用手动验证成功登陆了，谢谢回复
